catalogs.yaml

```diff
@@ -0,0 +1,3 @@
+# List of Software Template catalogs to setup with RHDH
+catalogs:
+  - https://github.com/redhat-ai-dev/ai-lab-template/blob/main/all.yaml
```

Comment:
Keep in mind at this current point in time until my issue is complete for swapping to this installer in the templates this will not allow component creation.

Reply:
@Jdubrick Yeah you'll need to use your changes which can be done following [this](https://github.com/michael-valdron/ai-rhdh-installer/tree/feat/rhdh-config?tab=readme-ov-file#setting-catalogs-for-developer-hub-configuration), this is just the default list for when all pipeline definition and template changes are in.